Gemfile.lock

```diff
@@ -1,75 +1,75 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (6.0.2.2)
-      actionpack (= 6.0.2.2)
+    actioncable (6.0.3.2)
+      actionpack (= 6.0.3.2)
       nio4r (~> 2.0)
       websocket-driver (>= 0.6.1)
-    actionmailbox (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activejob (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
-      activestorage (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actionmailbox (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      activejob (= 6.0.3.2)
+      activerecord (= 6.0.3.2)
+      activestorage (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
       mail (>= 2.7.1)
-    actionmailer (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      actionview (= 6.0.2.2)
-      activejob (= 6.0.2.2)
+    actionmailer (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      actionview (= 6.0.3.2)
+      activejob (= 6.0.3.2)
       mail (~> 2.5, >= 2.5.4)
       rails-dom-testing (~> 2.0)
-    actionpack (6.0.2.2)
-      actionview (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actionpack (6.0.3.2)
+      actionview (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actiontext (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
-      activestorage (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actiontext (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      activerecord (= 6.0.3.2)
+      activestorage (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
       nokogiri (>= 1.8.5)
-    actionview (6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actionview (6.0.3.2)
+      activesupport (= 6.0.3.2)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activejob (6.0.2.2)
-      activesupport (= 6.0.2.2)
+    activejob (6.0.3.2)
+      activesupport (= 6.0.3.2)
       globalid (>= 0.3.6)
-    activemodel (6.0.2.2)
-      activesupport (= 6.0.2.2)
-    activerecord (6.0.2.2)
-      activemodel (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
-    activestorage (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activejob (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
+    activemodel (6.0.3.2)
+      activesupport (= 6.0.3.2)
+    activerecord (6.0.3.2)
+      activemodel (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
+    activestorage (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      activejob (= 6.0.3.2)
+      activerecord (= 6.0.3.2)
       marcel (~> 0.3.1)
-    activesupport (6.0.2.2)
+    activesupport (6.0.3.2)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.2)
+      zeitwerk (~> 2.2, >= 2.2.2)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
-    autoprefixer-rails (9.7.5)
+    autoprefixer-rails (9.8.6.1)
```

Comment:
### [autoprefixer-rails](https://github.com/ai/autoprefixer-rails)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 9.7.5 → 9.8.6.1

[_[change-log](https://github.com/ai/autoprefixer-rails/blob/master/CHANGELOG.md), [source-code](https://github.com/ai/autoprefixer-rails)_]





<details>
<summary>Commits</summary>

A change of **32** commits. See the full changes on [the compare page](https://github.com/ai/autoprefixer-rails/compare/9.7.5...9.8.6.1).

These are the first 10 commits:
- (ec1ff23) [Update autoprefixer.js with reverted strech fix](https://github.com/ai/autoprefixer-rails/commit/ec1ff23a6243f2022d69243a3ee0c19388601177)
- (6506a68) [Release 9.7.6 version](https://github.com/ai/autoprefixer-rails/commit/6506a68a4a29d5bde968def5baefe67ffeee6e69)
- (8b9d350) [Update autoprefixer.js with :placeholder-shown](https://github.com/ai/autoprefixer-rails/commit/8b9d350afff8dd42da6dbd062befd0a6d4ce03b8)
- (343c018) [Add Ruby 2.7 on CI](https://github.com/ai/autoprefixer-rails/commit/343c0183d3dcb4c6e933ed2d940b2aeb318bd398)
- (94c7c56) [Update autoprefixer.js without debug code](https://github.com/ai/autoprefixer-rails/commit/94c7c569f1616c1167a69782bc9652f2b69c1b3c)
- (ff17f4f) [Release 9.8 version](https://github.com/ai/autoprefixer-rails/commit/ff17f4f7ea4a25bb04d8a38c94e1e064f10df0c7)
- (da78563) [Update autoprefixer.js with new color output library](https://github.com/ai/autoprefixer-rails/commit/da785638ddf86bd64223e7549ff16e9b435df32b)
- (86c7e74) [Release 9.8.1 version](https://github.com/ai/autoprefixer-rails/commit/86c7e74e678e45cffa313d7ebec8bcfdcd73d1a0)
- (daa9cd2) [Update autoprefixer.js with Can I Use update](https://github.com/ai/autoprefixer-rails/commit/daa9cd25ae6c4d8e134e0a2bd0bba2dfcbe7508d)
- (8818a71) [Release 9.8.2 version](https://github.com/ai/autoprefixer-rails/commit/8818a71d6651c24af368a2b4a7d27d2882515216)


</details>